habitat-lab/habitat/tasks/rearrange/sub_tasks/cat_nav_to_obj_task.py

```diff
@@ -13,23 +16,52 @@
 
 @registry.register_task(name="CatNavToObjTask-v0")
 class CatDynNavRLEnv(DynNavRLEnv):
+    def __init__(self, *args, config, dataset=None, **kwargs):
+        super().__init__(
+            config=config,
+            *args,
+            dataset=dataset,
+            **kwargs,
+        )
+        self._receptacle_semantic_ids: Dict[int, int] = {}
+
+    @property
+    def receptacle_semantic_ids(self):
+        return self._receptacle_semantic_ids
+
+    def reset(self, episode: Episode):
+        obs = super().reset(episode)
+        self._cache_receptacles()
+        return obs
+
+    def _cache_receptacles(self):
```

Comment:
In the future, this is probably good information to expose from the simulator. I think `habitat.datasets.rearrange.samplers.receptacle.find_receptacles` might also expose this same functionality?